pipeline.yaml

```diff
@@ -80,18 +87,25 @@ steps:
     # be registered to the MLflow Model Registry
     allow_non_validated_model: false
   predict:
+    #
+    # FIXME::REQUIRED: Specify the output format of the scored data from predict
```

Comment:
Can we make all batch scoring usage as optional? I don't think user has to define these before a model is trained.